backend/twitterbot/top_contributors.py

```diff
@@ -0,0 +1,40 @@
+import collections
+from datetime import datetime, timedelta
+
+from tournesol.models.comparisons import Comparison
+from tournesol.models.poll import DEFAULT_POLL_NAME
+from tournesol.models.ratings import ContributorRating
+
+
+def get_previous_month_top_public_contributor():
+    """Return the top public contributor of the previous month."""
+
+    now = datetime.now()
+    last_month = datetime(now.year, now.month, 1) - timedelta(days=15)
```

Comment:
Should we use `timedelta(days=30)`?

Reply:
No, it's a trick!
With a time delta of 30 days we could get e.g.:
- May instead of April by asking on the 31st of May
- January instead of February by asking on the 1st of March

Reply:
Oh I see.

Should we make the month a parameter of the function then? For instance we can ask for the top contributors of may, and we are sure the function will return the same result independently of the current time. I think it's better to rely a maximum on predictable functions, but as I don't want to give you more work we could do this in two PR : )

Reply:
Yes we could, but as we want to automatize that, we will have the same issue in the function which call this one. 
In the issue @lfaucon propose that we create automatically a PNG (I imagine the 1st of each month for the previous month) and we use this PNG to create the tweet:

> I'd propose to have a cronjob that generates the right png in the server's static file such that it is easily accessible
I think it is easiest to start implementing this feature inside the Django twitter app for now. If we get more data-analytics close to the django app, we would later create a separate app for it.

Reply:
Ok I understand now.

As long as we don't need to run the function for specific months, using the `timedelta(days=15)` will work perfectly :ok_hand: